src/applications/dhp-connected-devices/components/ConnectedDevicesContainer.jsx

```diff
@@ -36,7 +38,11 @@ export const ConnectedDevicesContainer = () => {
       try {
         const headers = { 'Content-Type': 'application/json' };
         const response = await apiRequest(FETCH_CONNECTED_DEVICES, { headers });
-        setConnectedDevices(response?.devices);
+        // console.log('responses from call', response);
+        if (response?.connectionAvailable) {
+          setConnectedDevices(response?.devices);
```

Comment:
don't really need the `?` in `response?` on this line since we know it's not null based on the previous line